subprojects/docs/src/docs/userguide/migration/upgrading_version_7.adoc

```diff
@@ -51,6 +51,14 @@ Prior to this version, a dependency substitution target could not be excluded fr
 This was caused by checking for exclusions prior to performing the substitution.
 Now Gradle will also check for exclusion on the substitution result.
 
+[[deprecate_unreadable_output]]
```

Comment:
This should be `declare_unreadable_output`, no?

Reply:
I renamed this to `declare_unreadable_input_output`.